r-package/grf/tests/testthat/test_causal_forest_tuning.R

```diff
@@ -7,37 +7,33 @@ test_that("causal forest tuning decreases prediction error", {
   TAU <- 0.1 * (X[, 1] > 0)
   Y <- TAU * (W - 1 / 2) + 2 * rnorm(n)
 
-  forest <- causal_forest(X, Y, W, num.trees = 400, min.node.size = 1, tune.parameters = FALSE)
+  forest <- causal_forest(X, Y, W, num.trees = 400, min.node.size = 1, tune.parameters = "none")
   preds <- predict(forest)
   error <- mean((preds$predictions - TAU)^2)
 
-  tuned.forest <- causal_forest(X, Y, W, num.trees = 400, tune.parameters = TRUE)
+  tuned.forest <- causal_forest(X, Y, W, num.trees = 400, tune.parameters = "all")
   tuned.preds <- predict(tuned.forest)
   tuned.error <- mean((tuned.preds$predictions - TAU)^2)
 
   expect_true(tuned.error < error * 0.75)
 })
 
-test_that("causal forest tuning only cross-validates null parameters", {
-  p <- 6
+test_that("causal forest forest tuning tunes each parameter", {
+  p <- 4
```

Comment:
Just checking: are these test modifications on purpose?

Reply:
Ah, yes this is basically just replacing the above test (only testing that a parameter is tuned), and I just set p=4 because it was faster